client/components/housing/HouseCard.tsx

```diff
@@ -96,31 +96,31 @@ const ClubCardTag = (tag: string) => {
   }
 }
 
-interface ClubCardProps {
-  club: Club
+interface HouseCardProps {
+  House: House
 }
 
-export const ClubCard = ({ club }: ClubCardProps) => {
-  const { id, name, description, links, tags } = club
+export const HouseCard = ({ House }: HouseCardProps) => {
+  const { id, name, description } = House
   const router = useRouter()
 
   const handleClick = () => {
-    router.push({ pathname: `/clubs/${id}` })
+    router.push({ pathname: `/Houses/${id}` })
```

Comment:
I think this is the actual correct pathname given what you have in `NavBar.tsx`

```suggestion
    router.push({ pathname: `/housing/${id}` })
```

Reply:
Good catch, changed.